spec/support/fog_helper.rb

```diff
@@ -0,0 +1,4 @@
+Fog.mock!
+Fog.credentials_path = Rails.root.join('config/fog_credentials.yml')
+connection = Fog::Storage.new(:provider => 'AWS')
+connection.directories.create(:key => 'my_bucket')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.